cluster/swarm/cluster.go

```diff
@@ -298,7 +298,7 @@ func (c *Cluster) RemoveImages(name string) ([]*dockerclient.ImageDelete, error)
 	errs := []string{}
 	var err error
 	for _, e := range c.engines {
-		for _, image := range e.Images() {
+		for _, image := range e.Images(true) {
```

Comment:
Correct me if I am wrong. The default value is `false` so why `true` is used here?